intro.tex

```diff
@@ -36,33 +36,21 @@ \subsection{Early Science scenarios } \label{ssec:scenarios}
 
 \begin{itemize}
 \item \textbf{Scenario A}:
-The full commissioning plan comprising system optimization and science validation is successfully executed as planned. 
-Rubin Operations then carries out an Operations Rehearsal and Operations Readiness Review (ORR) to effectively conduct a \textit{full dress rehearsal} of science operations and demonstrate the readiness of the Operations team to execute the 10-year survey. 
-Science-grade data collected during the commissioning System Optimization period and subsequent Science Validation Surveys, \S~\ref{sec:commissioning}, is reprocessed to produce the final Data Preview, DP2, which will be released 6 months following the completion of the Science Validation Surveys.
+Rubin Observatory is ready to execute the 10-year LSST at the completion of the construction project. 
+The operations team may decide to first conduct a \textit{full dress rehearsal} of science operations to demonstrate team readiness prior to commencing execution of the LSST.
+We expect this to take a few days to no longer than two weeks. 
 
 \item \textbf{Scenario B}:
-On-sky time in commissioning is reduced as the construction work  draws to an end, resulting in the SV surveys not being completed prior to the end of the construction phase.
-The Operations team would spend up to 3 months prior to commencing the 10-year LSST survey completing any remaining SV Survey observations.
-As per Scenario A, data collected during commissioning and the SV Surveys is reprocessed to produce DP2 and an Operations Readiness Review carried out to demonstrate readiness to execute the 10-year survey. 
+Prior to commencing the 10-year LSST, the operations team decides to spend up to a maximum of 2 months collecting more on-sky data to complement and extend the datasets collected during commissioning. 
```

Comment:
I'm nervous about this phrasing "Depending on the state of the as-built observatory system handed over at ORR" - it sort of implies that the as-built system handed over at ORR might not be ready to start the LSST survey. This is the sort of implication that the rewording is trying to avoid by simply stating that we might decide to collect more data prior to starting the survey for the 2 reasons stated.

Reply:
I tend to agree with Leanne's suggestion here of a more general phrasing. There might be compelling scientific motivation that would enable more and higher impact early science by taking additional data prior to the start of normal 10-year LSST.

Reply:
After I made the suggestion I ran this text past Victor and @smr456 , but have not heard back yet. You can go with the more ambiguous text if you prefer, no problem - for this document I think what you had is fine.

Reply:
Thanks Phil (note I hd reviewed this text with Bob before he left on vacation). I would rather remain ambiguous at this time - we retain greater flexibility this way.